go.mod

```diff
@@ -3,84 +3,82 @@ module github.com/stlaz/psachecker
 go 1.17
```

Comment:
do we want to bump the go version as well?

Reply:
done